src/test/java/com/transferwise/openbanking/client/api/registration/RestRegistrationClientTest.java

```diff
@@ -111,13 +143,11 @@ void registerClientThrowsApiCallExceptionOnApiCallFailure() {
         mockAspspServer.verify();
     }
 
-    @ParameterizedTest
-    @MethodSource("argumentsForRegisterClientTest")
-    void updateRegistration(boolean registrationUsesJoseContentType, String expectedContentType)
-        throws Exception {
-
+    @Test
+    void updateRegistration() throws Exception {
```

Comment:
I decided to refactor the update registration tests to separate out the specific tests for the different content type values and for the different scope values, rather than having a single test with lots of parameters.